lib/imperative-test.js

```diff
@@ -274,6 +274,28 @@ class ImperativeTest extends Test {
     this._check('doesNotThrow', actual => !actual, actual, undefined, message);
   }
 
+  mustNotCall(fn, name = 'anonymous') {
+    return this.mustCall(fn, 0, name);
+  }
+
+  mustCall(fn, count = 1, name = 'anonymous') {
+    let counter = 0;
+    this.on('beforeDone', () => {
+      this._check(
+        'mustCall',
```

Comment:
You are using that same `type` value for `mustNotCall`, I'd say it should have different `type`.

Reply:
That way, we can also specify message more precisely, so instead of `'function was called N times but was expected to be called 0 times'` it can be `'function was called N times but was not expected to be called at all'`, which is a more human readable format, in my opinion.